.github/workflows/deploy.yml

```diff
@@ -89,7 +90,7 @@ jobs:
       # Configure
       - name: Enable plugins
         # missing plugins: none
-        run: $SSH "$TUTOR plugins enable mfe forum notes xqueue discovery ecommerce indigo"
```

Comment:
Watch out, you removed the indigo plugin here. I think you need to rebase on top of the latest master version.